frontend/android/src/main/java/io/github/droidkaigi/confsched2019/App.kt

```diff
@@ -81,6 +81,9 @@ open class App : DaggerApplication() {
     }
 
     private fun setupCrashlytics() {
+        if (!BuildConfig.IS_BUILT_ON_CI) {
+            return
+        }
         Fabric.with(this, Crashlytics())
```

Comment:
The approach looks correct but Crashlytics needs to be initialized anyway, and it can disable itself from the code or AndroidManifest metadata. 
Could you please use `Crashlytics.Builder#disabled` or *firebase_crashlytics_collection_enabled metadata*?

Reply:
Thank you for your review. 
I got it. I will try it !